src/main/java/com/statistic/deputies/repository/DeputatRepository.java

```diff
@@ -25,4 +25,8 @@
     @Query(value = "SELECT d FROM Deputat d WHERE d.endWork IS NOT NULL "
             + "ORDER BY d.endWork - d.startWork")
     List<Deputat> getDeputiesWithShortestActiveTerms();
+
+    List<Deputat> findByEduContaining(String university);
```

Comment:
```suggestion
    List<Deputat> findByEducationContaining(String university);
```

Reply:
Renamed method and corresponding variable